pkg/login/social/social.go

```diff
@@ -63,6 +63,7 @@ type OAuthInfo struct {
 	AutoLogin               bool              `mapstructure:"auto_login" toml:"auto_login"`
 	ClientId                string            `mapstructure:"client_id" toml:"client_id"`
 	ClientSecret            string            `mapstructure:"client_secret" toml:"-"`
+	DisableHDValidation     bool              `mapstructure:"disable_hd_validation" toml:"disable_hd_validation"`
```

Comment:
Please don't put this into the OAuthInfo if it only applies to the Google integration. Rather please create a custom key like: https://github.com/grafana/grafana/blob/3f2eb8bd6a57d1449ae7e6a76a98e69a58d5e8d5/pkg/login/social/connectors/azuread_oauth.go#L82

Reply:
I've moved the `DisableHDValidation` to the connector as requested. PTAL